circle.yml

```diff
@@ -134,7 +142,8 @@ workflows:
     jobs:
       - lint
       - build
```

Comment:
Perhaps rename this to `build-cx11` for clarity?

Reply:
Actually does this mean we're not building in pure C mode on the CI anymore?

Reply:
Ah ok the toolchain configuration is for `CMAKE_CXX_STANDARD`. Is it worth separately to have `CMAKE_C_STANDARD` and build for C99 and C11? I don't see any worth going below C99 in any case.

Reply:
Note: this was discussed in #19 and is implemented in the tests.

I think it just confusing that CI tests C++ while some of the tests check C standard. Would be nice to document.

Reply:
Created an issue to track this.